### PR TITLE
Re-generate with new version of components-generator

### PR DESCRIPTION
**Pull request by pnedelko**

### CODE REVIEW

File: lib/triggers/trigger.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ function processTrigger(msg, cfg, snapshot, incomingMessageHeaders, tokenData) {
     if(skipSnapshot){
       return newElement.data; 
     } else {
-      await dataAndSnapshot(newElement,snapshot,snapshotKey, 'startDate', this);
+      await dataAndSnapshot(newElement,snapshot,snapshotKey, , this);
```
Comment (pnedelko) [inline lib/triggers/trigger.js]: Looks suspicious

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "version": "4.8.0",
+    "version": "4.7.0",
```
Comment (pnedelko) [inline package.json]: Should we update templates in the generator?

Comment (weberjm) [reply package.json]: So, the generator looks correct. The current Sage People version is 4.8, as generated before. I'm not sure how you got version 4.7 of the API to build off of https://developer.sage.com/people/reference/

File: lib/triggers/trigger.js (hunk 1 above)
Comment (weberjm) [reply lib/triggers/trigger.js]: Hmm, looks like the generator is written to use human input on the command line to determine what the snapshot key is: https://github.com/openintegrationhub/component-generator/tree/391ed9dfca1ed6c5e8ba0d39dc2db7442a444fb6#command-line-arguments-and-options-and-path-build

Comment (pnedelko) [reply lib/triggers/trigger.js]: Yes, I've seen that. I believe there are 2 options:
1. Make the snapshot param required
2. Add a default value for this param

What do you think?

File: package.json (hunk 2 above)
Comment (pnedelko) [reply package.json]: seems like I took the wrong api description file. Re-generated with 4.8.

**Pull request merged**